diff --git a/klippy/extras/aht10.py b/klippy/extras/aht10.py
index ead58fbce..b0636f378 100644
--- a/klippy/extras/aht10.py
+++ b/klippy/extras/aht10.py
@@ -37,7 +37,7 @@ class AHTBase:
         self.reactor = self.printer.get_reactor()
         self.i2c = bus.MCU_I2C_from_config(
             config, default_addr=I2C_ADDR, default_speed=100000)
-        self.report_time = config.getint('aht10_report_time', 30, minval=5)
+        self.report_time = config.getint('aht10_report_time', 30, minval=1)
         self.temp = self.min_temp = self.max_temp = self.humidity = 0.
         self.sample_timer = self.reactor.register_timer(self._sample_aht)
 
